dynamax/linear_gaussian_ssm/parallel_inference_test.py

```diff
@@ -12,86 +12,111 @@
 from dynamax.linear_gaussian_ssm.inference_test import flatten_diagonal_emission_cov
 
```

Comment:
Test cases are updated to check if the parallel inference can handle inputs.

Reply:
Also, the synthetic data for testing the time-varying case was too simplistic to capture some bugs while I was developing the code. So I updated the test case so that it has more time variation of parameters.